packages/phone-number-privacy/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celo/phone-number-privacy-common",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Can you rev the signer + combiner versions too and keep them in sync?

Reply:
I was thinking more about this and feel that it's fine if the versions diverge across all three libs. We may frequently role out combiner updates going fwd and will not want to constantly bump the signer version with no changes. Currently signer is on v1.0.3 and the combiner is on v1.0.0. Perhaps it makes most sense to keep the major versions in sync but not necessarily the minor/patch. WDYT?

Reply:
Ok works